src/test/java/Pages/Loginpage.java

```diff
@@ -3,12 +3,12 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 
-public class LoginPage extends BasePage{
+public class Loginpage extends Basepage {
     static By emailFieldLocator = By.cssSelector("[type='email']");
     static By passwordFieldLocator = By.cssSelector("[type='password']");
     static By submitButtonLocator = By.cssSelector("button[type='submit']");
 
-    public LoginPage(WebDriver givenDriver) {
+    public Loginpage(WebDriver givenDriver) {
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/226467717-5df6e22b-5e63-48c2-981a-d8ecb2531e01.png)

Hint

```
public class Loginpage extends Basepage {

  private By emailFieldLocator = By.cssSelector("[type='email']");
  private By passwordFieldLocator = By.cssSelector("[type='password']");
  private By submitButtonLocator = By.cssSelector("button[type='submit']");

  public Loginpage(WebDriver givenDriver) {
    super(givenDriver);
  }

  public void provideEmail(String email) {
    driver.findElement(emailFieldLocator).sendKeys(email);

  }

  public void providePassword(String password) {
    driver.findElement(passwordFieldLocator).sendKeys(password);
  }

  public void clickSubmitBtn() {
    driver.findElement(submitButtonLocator).click();

  }

  public void login() {
    provideEmail("andalia.nia@gmail.com");
    providePassword("te$t$tudent");
    clickSubmitBtn();
  }

}
```